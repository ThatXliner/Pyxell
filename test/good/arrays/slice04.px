
a = [x for x in 0..100]
print a[::4].length

z = -10
for x in a[10:z:27] do
    print x
for x in a[z+1:z+2:364] do
    print x
