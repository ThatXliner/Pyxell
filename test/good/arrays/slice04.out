26
10
37
64
92
